backend/Cargo.toml

```diff
@@ -43,3 +44,4 @@ tokio = { version = "1.36.0", features = [
     "time",
 ] }
 uuid = { version = "1.7.0", features = ["serde"] }
+gamma_rust_client = { git = "https://github.com/viddem/gamma.git", rev = "1235842562c9f3b50a68582dea13f66fbe93cf5e" }
```

Comment:
Small TODO here. We should switch to the cthit hosted gamma repo and not a fork when the [gamma pr](https://github.com/cthit/Gamma/pull/880) is merged

Reply:
Done!